core/test/validation_test.jl

```diff
@@ -186,13 +224,13 @@ end
     @test logger.logs[2].level == Error
     @test logger.logs[2].message ==
           "Fractional flow nodes must have non-negative fractions."
-    @test logger.logs[2].kwargs[:node_id] == 3
+    @test logger.logs[2].kwargs[:node_id].value == 3
     @test logger.logs[2].kwargs[:fraction] ≈ -0.1
     @test logger.logs[2].kwargs[:control_state] == ""
     @test logger.logs[3].level == Error
     @test logger.logs[3].message ==
           "The sum of fractional flow fractions leaving a node must be ≈1."
-    @test logger.logs[3].kwargs[:node_id] == 7
+    @test logger.logs[3].kwargs[:node_id].value == 7
```

Comment:
A better test would be to remove the `.value` and `== NodeID(...)`, since that explicitly tests if it is a NodeID.